### PR TITLE
Put application in DSS section as last section and add TGB

### PR DESCRIPTION
**Pull request by FlorianJacta**

### CODE REVIEW

**Review by AlexandreSajus — CHANGES_REQUESTED** [review]
Good initiative! Just one tbg-markdown selector is broken (I think missing indentation):
image

File: docs/tutorials/articles/dataiku/index.md [hunk 1]
```diff
@@ -187,36 +106,96 @@ Create your own Taipy application to visualize all this information. It can
...
+            ),
+            x="Name",
+            y="Value",
+            type="bar",
+        )
+
+    Gui(md).run()
+    ```
+
+    [Download the code](./src/metrics_visualization.py){: .tp-btn target='blank' }
```
Comment (AlexandreSajus) [inline docs/tutorials/articles/dataiku/index.md]: One last thing, this link is wrong, it should link to the tgb version

**Review by AlexandreSajus — CHANGES_REQUESTED** [review]
One last thing, wrong link in line 197

File: docs/tutorials/articles/dataiku/index.md (hunk 1 above)
Comment (FlorianJacta) [reply docs/tutorials/articles/dataiku/index.md]: Thanks

**Review by AlexandreSajus — APPROVED** [review]
Looks good to me!

**Pull request merged**